src/routes/transactions/transactions-history.controller.spec.ts

```diff
@@ -298,55 +305,92 @@ describe('Transactions History Controller (Unit)', () => {
-    );
-    moduleTransaction.txType = 'MODULE_TRANSACTION';
-    multisigTransaction.txType = 'MULTISIG_TRANSACTION';
-    incomingTransaction.txType = 'ETHEREUM_TRANSACTION';
-    const tokenResponse = getJsonResource(
-      'multisig-transactions/erc20/token-source-data.json',
-    );
-    const safe = getJsonResource(
-      'incoming-transfers/native-coin/safe-source-data.json',
-    );
-    const transactionHistoryBuilder = {
-      count: 40,
-      next: `${chainResponse.transactionService}/api/v1/safes/${safeAddress}/all-transactions/?executed=false&limit=10&offset=10&queued=true&trusted=true`,
-      previous: null,
-      results: [moduleTransaction, multisigTransaction, incomingTransaction],
-    };
+    const moduleTransaction = moduleTransactionBuilder()
+      .with('executionDate', new Date('2022-12-14T13:19:12Z'))
+      .with('isSuccessful', true)
+      .with('safe', '0x4127839cdf4F73d9fC9a2C2861d8d1799e9DF40C')
+      .with('data', null)
+      .with('operation', 0)
+      .with('value', faker.number.int({ min: 1 }).toString())
+      .with('moduleTransactionId', 'i5a6754140f0432d3b')
+      .build();
+    const multisigTransactionToAddress = faker.finance.ethereumAddress();
+    const multisigTransactionValue = faker.string.numeric();
+    const multisigTransaction = multisigTransactionBuilder()
+      .with('safe', '0xC22de4CeeAFd9436d20A9b18C5970D680D3498c8')
+      .with('value', '0')
+      .with('operation', 0)
+      .with('safeTxGas', 0)
+      .with('executionDate', new Date('2022-11-16T07:31:11Z'))
+      .with('submissionDate', new Date('2022-11-16T07:29:56.401601Z'))
+      .with('safeTxHash', '0x31d44c67')
+      .with('isExecuted', true)
+      .with('isSuccessful', true)
+      .with('origin', null)
+      .with(
+        'dataDecoded',
+        dataDecodedBuilder()
+          .with('method', 'transfer')
+          .with('parameters', [
+            dataDecodedParameterBuilder()
+              .with('name', 'to')
+              .with('type', 'address')
+              .with('value', multisigTransactionToAddress)
+              .build(),
+            dataDecodedParameterBuilder()
+              .with('name', 'value')
+              .with('type', 'uint256')
+              .with('value', multisigTransactionValue)
+              .build(),
+          ])
+          .build(),
+      )
+      .with('confirmationsRequired', 2)
+      .with('confirmations', [
+        confirmationBuilder().build(),
+        confirmationBuilder().build(),
+      ])
+      .build();
+    const safe = safeBuilder()
+      .with('address', '0x4127839cdf4F73d9fC9a2C2861d8d1799e9DF40C')
+      .build();
+    const nativeTokenTransfer = nativeTokenTransferBuilder()
+      .with('executionDate', new Date('2022-08-04T12:44:22Z'))
+      .with('to', safe.address)
+      .with('transferId', 'e1015fc6905859c69')
+      .build();
+    const transfer: any = nativeTokenTransferToJson(nativeTokenTransfer);
```

Comment:
Nit: can we use the correct type here instead of `any`?

Reply:
Changed in [0450df9](https://github.com/safe-global/safe-client-gateway-nest/pull/503/commits/0450df9f6fa71b7b2f5bd8b364e0fa0aedbdb2f3)